packages/algorithm/package.json

```
@@ -26,9 +26,9 @@
     "api": "api-extractor run --local --verbose",
     "build": "npm run build:src && rollup -c",
     "build:src": "tsc --build",
-    "build:test": "tsc --build tests && cd tests && webpack",
+    "build:test": "npm run clean:test && tsc --build tests && cd tests && rollup -c",
     "clean": "rimraf ./lib && rimraf *.tsbuildinfo && rimraf ./types && rimraf ./dist",
-    "clean:test": "rimraf tests/build",
+    "clean:test": "rimraf tests/lib && rimraf tests/tsconfig.tsbuildinfo",
```

FYI you can pass multiple targets to `rimraf`:

```suggestion
    "clean:test": "rimraf tests/lib tests/tsconfig.tsbuildinfo",
```